app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -476,7 +480,7 @@ def attributes_for_log
     attributes["purchid"] = field_1
     attributes["saledate"] = saledate
 
-    attributes["noint"] = 2 if field_6 == 1
+    attributes["noint"] = 2 if buyer_not_interviewed?
```

Comment:
Do we want to set `noint` to be 1 if the buyer was interviewed?

Reply:
this has made me realise I think this was just wrong, it should be set to the same as field_6